docs/source/cli_help/status.rst

```diff
@@ -0,0 +1,63 @@
+status
+---------
+Usage:
+::
+    git machete s[tatus] [--color=WHEN] [-l\|--list-commits] [-L\|--list-commits-with-hashes] [--no-detect-squash-merges]
+
+Displays a tree-shaped status of the branches listed in the definition file.
+
+Apart from simply ASCII-formatting the definition file, this also:
+
+    * colors the edges between upstream (parent) and downstream (children) branches:
+
+      - red edge means that the downstream branch tip is not a direct descendant of the upstream branch tip,
```

Comment:
Does rst support actual colors?

Reply:
It looks like that not directly. I'm investigating some workaround to add them.

Reply:
Would be nice, albeit non-critical ofc

Reply:
I've added some coloring, but I didn't manage to get it to work in code blocks and it's a bit dirty solution. I'm adding roles (as I understand it's the recommend way of doing colored text), but I'm making it work via this:
```
.. raw:: html

    <style> .green {color:green} </style>
    <style> .grey {color:dimgrey} </style>
    <style> .red {color:red} </style>
    <style> .yellow {color:yellow} </style>

.. role:: green
.. role:: grey
.. role:: red
.. role:: yellow
```

Html style can be extracted to separate files and it would be cleaner, but I don't want to prolong this review much longer, so I will call it a day at that and create enhancement issue to move it to appropriate location. I'm not doing it right now, because it requires some investigation on how to expand html style in sphinx, which directory should it be added to, will they collide with some other ones etc.

Reply:
Still, looks very good 🎨